sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -5,6 +5,7 @@
 from typing import Any, Union, Optional, TYPE_CHECKING
 import logging
 from weakref import WeakSet
+from typing_extensions import Literal
```

Comment:
As you mentioned in a comment elsewhere - I think this will need to be:
```python
try:
    from typing import Literal
except ImportError:
    from typing_extensions import Literal
```

Reply:
this causes a mypy error, same as this one: https://github.com/python/typeshed/issues/3500

They mention checking python version as the solution. I'm not sure if that's what we want to do?

Reply:
If we add the setup.py amendment above (which we probably should - we don't want unnecessary dependencies) then typing_extensions wont be available in <3.8.
As linked to within the issue you linked to - Mypy should support/understand Python version checks.

Reply:
sounds good, thank you!

Reply:
It looks like there's still an issue with mypy, but I think this is because this mypy check in the pipeline is using Python 3.7.

I can ask Scott about this.